### ISSUE TITLE
[A11Y] URL field label is hidden

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#3421]
**Browser / Version**: Firefox Nightly 81.0a1 (2020-07-27), Chrome 84.0.4147.89 
**Operating System**: Windows 10 Pro

**Steps to Reproduce**:
1. Navigate to https://webcompat.com/issues/new
2. Observe URL field label.

**Expected Behavior:**
Label is visible.

**Actual Behavior:**
Label is hidden.

**Notes:**
1. Screenshot attached.
2. According to "Inspector>Accessibility" tab, "Form elements should have a visible text label".
![image](https://user-images.githubusercontent.com/12184325/88671712-4e70d500-d0ef-11ea-8185-25ebfd42f9fe.png)

**Watchers:**
@softvision-oana-arbuzov
@cipriansv

**Affected area:**
```html
<div class="form-text form-element js-Form-group form-label-message with-validation-icons is-validated js-no-error">
	<label for="url" class="form-label is-hidden">Site URL (mandatory)</label>
	<div class="input-wrapper">
		<input class="form-field text-field required" id="url" name="url" pattern="" placeholder="e.g. https://example.com" required="" type="url" value="">
	</div>
</div>
```
![image](https://user-images.githubusercontent.com/12184325/88671180-a8bd6600-d0ee-11ea-8ad6-b20bc9eae0a3.png)

**Fixing proposal:** 
Changing `label class="form-label is-hidden"` to `label class="form-label is-visible"` the A11Y error is fixed.
![image](https://user-images.githubusercontent.com/12184325/88671423-f508a600-d0ee-11ea-974e-c1dc009c4391.png)

**Comment by miketaylr** [comment]
We have a "fake label" that's visible: "Please enter the URL where you encountered the web compatibility issue. "

Let me try to just make that the `<label>`, then we can get rid of the hidden one.

**Comment by softvision-oana-arbuzov** [comment]
Regarding the `"fake label" that's visible: ` , this is fixed.

```html
<div class="form-text form-element js-Form-group form-label-message with-validation-icons">
	<div class="input-wrapper">
		<input class="form-field text-field required" id="url" name="url" pattern="" placeholder="e.g. https://example.com" required="" type="url" value="">
	</div>
</div>
```

**Tested with:**
Browser / Version: Firefox Nightly 82.0a1 (2020-09-17)
Operating System: Windows 10 Pro
